src/libstd/keyword_docs.rs

```diff
@@ -387,10 +387,12 @@ mod extern_keyword {}
 //
 /// A value of type [`bool`] representing logical **false**.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// Logically `false` is not equal to [`true`].
+///
+/// See the documentation for [`true`] for more information.
 ///
+/// [`true`]: keyword.true.html
 /// [`bool`]: primitive.bool.html
```

Comment:
These should resolve automatically with intra-doc links.
```suggestion
```

Reply:
It makes the build fail when I remove them.

Reply:
Hmm, that seems like a bug in intra-doc links ... I'll investigate and get back to you.

Reply:
The warning is
```
warning: `[true]` cannot be resolved, ignoring it.
 --> prim.rs:1:6
  |
1 | /// [`true`] and [`bool`]
  |      ^^^^^^ cannot be resolved, ignoring
```
The bug is here: https://github.com/rust-lang/rust/blob/fe98de2a8e3941bb33d1149c3717d8ea0744e39c/src/librustdoc/passes/collect_intra_doc_links.rs#L1124
It only checks for types, not for values. I'll go through the list of keywords and add them.

Reply:
Oh yeah sorry, I should have provided the failing output.

Reply:
This turns out it will not be simple to fix ... you can still remove the link for `bool` but the other will have to be a manual link for now. https://github.com/rust-lang/rust/issues/74515

Reply:
Done for the `bool` link.